components/engine/src/rocks/util/config.rs

```diff
@@ -116,6 +116,24 @@ pub mod compression_type_level_serde {
     }
 }
 
+#[derive(Copy, Clone, Debug, PartialEq, Serialize, Deserialize)]
+#[serde(rename_all = "kebab-case")]
+pub enum BlobRunMode {
```

Comment:
Do we need to implement serializer and deserializer for `BlobRunMode`, like `DBCompressionType` above?

Reply:
No, what's above is for serializing DBCompressionType array.

Reply:
So how does, e.g. `blob-run-mode = "normal"` serialize into numeric value? Since `DBTitanDBBlobRunMode` is a numeric enum.

Reply:
All done by macro from `serde` I think.

Reply:
wow. looks like magic to me.